platforms/documentation/docs/src/docs/userguide/dep-man/understanding-dependency-resolution/dependency_resolution_model.adoc

```diff
@@ -0,0 +1,266 @@
+// Copyright (C) 2024 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+:metadata-file-spec: https://github.com/gradle/gradle/blob/master/platforms/documentation/docs/src/docs/design/gradle-module-metadata-latest-specification.md
+
+[[understanding_dependency_resolution_model]]
+= Understanding the Dependency Resolution Model
+
+This chapter explains how dependency resolution works within Gradle.
+After learning how to declare <<declaring_dependencies.adoc#one-declaring-dependencies,dependencies>> and <<dependency_locking.adoc#locking-versions,specify the versions>> your project should use, the next step is understanding how these declarations are combined during the dependency resolution process.
+
+Dependency resolution happens in two key phases, repeated until the entire dependency graph is constructed:
+
+1. **Conflict Resolution**:
+* When a new dependency is introduced, Gradle resolves any conflicts to determine the version that should be added to the graph.
+* Gradle will apply its conflict resolution rules (e.g., "latest version wins" or custom version resolution strategies) to determine the version to be used.
+2. **Dependency Metadata Retrieval**:
+* Once a specific dependency (a module with a version) is included in the graph, Gradle retrieves its metadata (such as POM, Ivy, or module metadata) to:
+** add its own dependencies (transitives) to the graph.
+** understand the available variants of that dependency.
+
+This process continues until the entire dependency tree is resolved.
+
+image::dependency-resolution-model.png[]
+
+[[sec:conflict-resolution]]
+== Phase 1. Conflict resolution
+
+When performing dependency resolution, Gradle handles two types of conflicts:
+
+A. **Version conflicts**: Which occur when multiple dependencies request the same dependency but with different versions.
+Gradle must choose which version to include in the graph.
+B. **Implementation / Capability conflicts**: Which occur when the dependency graph contains different modules that provide the same functionality or _capability_.
+Gradle resolves these by selecting one module to avoid duplicate implementations.
+
+The dependency resolution process is highly customizable and many APIs can influence the process.
+
+[[sec:version-conflict]]
+=== A. Version conflicts
+
+A version conflict occurs when two components:
```

Comment:
These are only two basic scenarios, but not the only ones.  I would suggest rewording it a bit so that it is not a full list

Reply:
Fixed.